interchange/DeviceResources.capnp

```diff
@@ -209,7 +229,8 @@ struct Device {
   }
 
   struct Node {
-    wires    @0 : List(WireIdx) $wireRef();
+    wires      @0 : List(WireIdx) $wireRef();
+    nodeTiming @1 : NodeTimingIdx $nodeTimingRef();
```

Comment:
It's notable that attaching the delay to nodes does mean we are permanently constrained to a representation where every sink on a node has the same delay. This is not strictly true for iCE40, but can probably be mitigated to some extent using pip delays.

It's _probably_ a reasonable enough assumption for place and route purposes, and certainly carries less overhead compared to putting delays on tile-wires only, but I think it's worth thinking this through and making sure we are happy with it before we commit to something that might be hard to change down the line.